content/source/docs/cloud/api/modules.html.md

```diff
@@ -59,11 +65,159 @@ $ curl \
   https://app.terraform.io/api/registry/v1/modules/my-cloud-org/consul/aws/versions
 ```
 
-## Publish a Module from a VCS
+### Sample Proxy Request (public module)
+
+List available versions for the `consul` module for the `aws` provider on the module registry published from the Github organization `my-gh-repo-org`:
+
+```shell
+$ curl https://registry.terraform.io/v1/modules/my-gh-repo-org/consul/aws/versions
+```
 
-`POST /registry-modules`
+The same request for the same module and provider on the Terraform Cloud module registry:
+
+```shell
+$ curl \
+  --header "Authorization: Bearer $TOKEN" \
+  https://app.terraform.io/api/registry/public/v1/modules/my-gh-repo-org/consul/aws/versions
+```
+
+## List Registry Modules for an organization
+
+`GET /organizations/:organization_name/registry-modules`
+
+Parameter            | Description
+---------------------|------------
+`:organization_name` | The name of the organization to list available modules from.
 
-Publishes a new registry module from a VCS repository, with module versions managed automatically by the repository's tags. The publishing process will fetch all tags in the source repository that look like [SemVer](https://semver.org/) versions with optional 'v' prefix. For each version, the tag is cloned and the config parsed to populate module details (input and output variables, readme, submodules, etc.). The [Module Registry Requirements](../../registry/modules/publish.html#requirements) define additional requirements on naming, standard module structure and tags for releases.
+Lists the modules that are available to a given organization. This includes the full list of publicly curated and private modules and is filterable.
+
+Status  | Response                                           | Reason
+--------|----------------------------------------------------|----------
+[200][] | [JSON API document][] (`type: "registry-modules"`) | The request was successful
+[404][] | [JSON API error object][]                          | Modules not found or user unauthorized to perform action
+
+### Filtering / Pagination
```

Comment:
It feels like we should use a table here, like we did with https://github.com/hashicorp/terraform-website/blob/master/content/source/docs/cloud/api/admin/organizations.html.md#query-parameters

Reply:
Moved to table